apps/slack/lambda/lib/errors/not-found.ts

```diff
@@ -1,7 +1,7 @@
-import { Exception } from './exception';
+import { Exception, ExceptionDetails } from './exception';
 
 export class NotFoundException extends Exception<string> {
```

Comment:
Should this be updated to `export class NotFoundException extends Exception<ExceptionDetails> {` since we are no longer passing in a string? Same thing in a few other files.

Reply:
good eye! adjusted!!